### DISCUSSION TITLE
Cannot subscribe to 'page' store on the server outside of a Svelte component, as it is bound to the current request via component context.

### DISCUSSION THREAD

**Discussion by aakash14goplani** [sveltejs/kit#10824] [Q&A]
I am getting following error on my root layout file.
> Cannot subscribe to 'page' store on the server outside of a Svelte component, as it is bound to the current request via component context. This prevents state from leaking between users. For more information, see https://kit.svelte.dev/docs/state-management#avoid-shared-state-on-the-server  

Since this error is thrown at root level, my application is unable to load. Here is the code snippet of my *+layout.svelte* file:
```svelte
<script lang="ts">
  import { page } from '$app/stores';
  import { browser } from '$app/environment';
  import Header from '$lib/components/Header.svelte';
  import Footer from '$lib/components/Footer.svelte';
  import { isLocaleLoaded } from '$lib/i18n/svelte-i18n';

  $: isSessionActive = browser && $page.data.session?.user?.email;
</script>

<div class="app">
  {#if browser && $isLocaleLoaded && $page.data.header}
    <Header header={$page.data.header} />
  {/if}

  <main>
    <slot />
  </main>

  {#if browser && $isLocaleLoaded && $page.data.footer}
    <Footer footer={$page.data.footer} />
  {/if}
</div>
```
I am unable to figure out how to fix this, could anyone suggest a fix?

Svelte Versions in use:
```json
"@sveltejs/adapter-vercel": "^3.0.3",
"@sveltejs/kit": "^1.25.1",
"svelte": "^4.2.1",
"svelte-check": "^3.5.2",
"svelte-media": "^0.1.5",
"svelte-preprocess": "^5.0.4",
"typescript": "^5.2.2",
"vite": "^4.4.9"
```

**Comment by david-plugge** [comment]
I can't replicate the issue right now as I'm not at my pc but I think your code should not error. Have you tried creating a minimal reproduction?

**Comment by BahaaZidan** [comment]
I had a similar error, all I did was update a few packages: (kit, svelte, vite, typescript, svelte-vite-plugin) and the problem went away.

**Comment by DePasqualeOrg** [comment]
I'm getting this error sporadically when I run SvelteKit in dev mode, but not in production. Is this anything to worry about?

**Comment by gmr458** [comment]
I deleted the `.svelte-kit` directory and ran the project again and the error went away

**Comment by aakash14goplani** [comment]
This happens with me when I update the version of dependencies and vite optimization happens. As mentioned in above comments, it happens in DEV mode only!
